ledger-tool/src/main.rs

```diff
@@ -1425,10 +1425,16 @@ fn main() {
                     .help("Include sysvars too"),
             )
             .arg(
-                Arg::with_name("exclude_account_data")
-                    .long("exclude-account-data")
+                Arg::with_name("no_accounts_contents")
+                    .long("no-accounts-contents")
```

Comment:
I think I like it when "verbose" means output more details about something _we're already doing_. It seems like the printing of accounts contents here is not something we're already doing, but conditionally doing (and kind of the whole point of the "accounts" subcommand). So that's why I ultimately chose a different impl here. Whatcha think?

Reply:
I considered the looping through all the accounts to be the thing we're already doing.

But I don't feel strongly about it. So if you do, I'll defer.
I do feel more strongly opposed to the double plural in `accounts-contents`, though. Can it just be `no-account-contents`?

Reply:
Will do! I went back and forth on that 's' too, hah!